README.md

```diff
@@ -27,16 +29,15 @@ The primary function of the bot is to establish a 1:1 linkage between a Discord
 3. Create a webhook named "Hubs" in the channels you want it to run in. It will use this webhook to bridge chat and
    send Hubs status updates.
 
-4. Try out the bot! Type `!hubs` in a channel the bot is in to see some things you can do. The bot will bind a Discord channel and a room if it sees the room URL in the topic of the channel.
+4. Try out the bot! Type `!hubs` in a channel the bot is in to see some things you can do. The bot will bridge a Discord channel and a room if it sees the room URL in the topic of the channel.
 
 ### Permissions
 
 The bot requires several permissions in order to work:
 
 - "Send messages," "Read messages," and "Embed links" are necessary in order to bridge between the Hubs room that is linked to a channel and the messages that are sent within the channel on Discord.
 - "Manage webhooks" is necessary in order for the bot to find and use a webhook for bridging chat.
-- "Manage channels" is necessary if you want to use the bot's channel topic management. If you don't want the bot to put
-  Hubs information in the topic, then you don't need it.
+- "Manage channels" is necessary in order for the bot to set the channel topic and bridge chat. **Note:** We do not ask for this permission globally when you add the bot to your server, instead we recommend you grant this permission to the bot in specific groups or channels.
```

Comment:
Something about this phrasing sounds a little odd -- e.g. we don't control what permissions the bot asks for when someone adds it to their server, the invite link specifies that. Also, we think that all the permissions should be granted to the specific groups or channels you want to use the bot in, not just this one.

I guess I would have written exactly what I wrote before, but if you like this better then you can change it if you want.

Reply:
Yeah, I was trying to approach this from the user's perspective for when we actually just give them a pre-generated link and point them to the readme. I don't imagine @misslivirose will tell people to generate their own link.

The bot does need manage_channels now. You can't run the `!hubs create` command without it. I did that intentionally since I figured bridging and permissions-binding ought to be a single operation, for simplicity.

Reply:
Okay, I agree this note I added is kinda in conflict with the very next paragraph in this section, but I suspect we will actually just hand a pre-generated link to community owners that will grant the bot global permissions for all required permissions except for "manage channels" and they will have to manually grant that last permission manually per channel/group. But feel free to change this accordingly once we decide on the final process for introducing communities @misslivirose. I will merge this as it is in the mean time.

Reply:
> I suspect we will actually just hand a pre-generated link to community owners that will grant the bot global permissions for all required permissions except for "manage channels" and they will have to manually grant that last permission manually per channel/group.

I'm surprised you think so, since you're the one who really disliked the idea of the bot being in our #private channel!

I think I endorse server owners adding all permissions by hand.